bzip2-sys/Cargo.toml

```diff
@@ -18,6 +18,9 @@ categories = ["external-ffi-bindings"]
 name = "bzip2_sys"
 path = "lib.rs"
 
+[features]
+no-stdio = []
```

Comment:
I think this was inverted? I think compiling without stdio should be the default because the crate doesn't need it and AFAIK no one using the crate today needs it?